mods/species/tritonian/species.dm

```diff
@@ -0,0 +1,28 @@
+/decl/species/human/tritonian
+	name = SPECIES_TRITONIAN
+	name_plural = "Tritonians"
+	description = "Of all human gene-adapts, the Tritonian genotype is probably the most wildly divergent from \
```

Comment:
As with Taj culture above, would like this to be generalized to 'A human-derived genotype designed for colonizing aquatic worlds' or something like that.